src/App.js

```diff
@@ -35,12 +35,11 @@ export class App extends Component {
 
   addFiles = async (files) => {
     const { doFilesWrite, doUpdateHash, routeInfo } = this.props
-
-    // Add the dropped files to the root
-    doFilesWrite('/', files)
-
+    const isFilesPage = routeInfo.pattern === '/files*'
```

Comment:
Just out of curiosity, what's the difference between `routeInfo.pattern === '/files*'` and `routeInfo.url.startsWith('/files')`?

Reply:
nothing much, just felt more explicit and fewer characters.